Makefile

```diff
@@ -1,7 +1,7 @@
 DOCKER_REVISION ?= snmp_exporter-$(USER)
 DOCKER_TAG = docker-push.ocf.berkeley.edu/snmp_exporter:$(DOCKER_REVISION)
 
-SNMP_EX_VERSION := v0.16.1
+SNMP_EX_VERSION := master
```

Comment:
Strongly recommend not doing this and instead use https://github.com/ocf/adelie to manage this version number